translations/frontend-component-header/src/i18n/transifex_input.json

```diff
@@ -33,5 +33,10 @@
   "header.user.menu.studio": "Studio Home",
   "header.user.menu.maintenance": "Maintenance",
   "header.label.courseOutline": "Back to course outline in Studio",
-  "header.label.search.nav": "Search content"
+  "header.label.search.nav": "Search content",
```

Comment:
[question] Do we need to add `EN` values to this plugin?

Reply:
as i remember, on Sigma project works as it is